docs/Home.md

```diff
@@ -1,3 +1,50 @@
-Welcome to the android-templates wiki! 
+Welcome to the android-templates wiki!
 
-More coming soon...
\ No newline at end of file
+This repository generates a new project based on our preferences, by running a simple script. 
+
+Ex: `./newproject.sh -t crt -p co.myproject.example -n "MyProjectExampleCoroutine"`
+
+This script must include all essentials by default, while optional features can be appended with flags. To prevent this repository from becoming a dumping playground, features can be rejected too.
+
+Planning to contribute? Let's have a look at the criteria:
+
+How do we decide if a feature is **essential**? 👍
+
+- Our team requires it to maintain code quality.
+- Our team requires it to maintain a good developer experience.
+- Our team requires it to increase the productivity of the team.
+- It is always used or configured in the projects we've worked on.
+- It is recommended to use or configure based on Google's standards.
+
+How do we decide if a feature is **optional**? 🚩
+
+- It is regularly used or configured in projects we've worked on.
+- It is regularly used or configured as an alternative to the default choice.
+
+How do we decide if a feature is **rejected**? 👎
+
+- It is barely used or configured in projects we've worked on.
+- It is barely used or configured as an alternative to the default choice.
+
+Let's have a look at some examples 🔎
+
+**Essential**:
+- [Detekt](https://github.com/detekt/detekt): Code smell analysis is important to maintain code quality.
+- [Hilt](https://developer.android.com/training/dependency-injection/hilt-android): Dependency injection provides a good developer experience.
+- [Firebase App Distribution](https://firebase.google.com/docs/app-distribution): Continuous delivery increases the productivity of the team.
+- [Timber](https://github.com/JakeWharton/timber): A logging library which is always used in all the projects we've worked on.
+- [Navigation Component](https://developer.android.com/guide/navigation/navigation-getting-started): A suite of libraries for in-app navigation which is recommended by Google.
```

Comment:
@Tuubz I think `Navigation Component` should be `Optional`, as not all projects need it 🤔

Reply:
I think of the opposite...if it is optional then you will have to provide the primary way to navigate in an application...I don't think the traditional fragment backstack manipulation is preferred right?

Reply:
@doannimble Could you perhaps give an example of which project didn't need it?

In my opinion, anything that Google recommends using should be `Essential`. So as they recommend using "[Single Activity with multiple fragments](https://youtu.be/9O1D_Ytk0xg?t=79)", navigation component comes with.

Reply:
@sleepylee @Tuubz  We can use fragment backstack with activity if it works and does not give us an issue. You guys can see at the end of the video [5:30](https://www.youtube.com/watch?v=9O1D_Ytk0xg&t=330s)  that if you're using Multi-Activity development and it works, stick with it. 

But if you're running into issues and bugs then to use Single Activity development.

The old way is not to be preferred, but it does not mean the `Navigation Component` is a must-have IMO 🙏

Reply:
we cannot leave the place for doubt like that... please read the description again on the essential part: the feature targetted as default or essential is the one that increase our dev experience and productivity. This is not a playground or fully crowd open source that we have to leave space for interpretation. We must own it and make the solid rule in the end.
As you can see, every project is now moving to navigation component; will you start the next project without it? Just answer that question to yourself then you might have the clear answer.

Reply:
@doannimble As for the android-template is for `us` to use it to initiate the android project that we're gonna work on it further. So I think this should combine the approach we usually use in the client project. 🙏🏻  So I think the `Navigation Component` is an Essential one. What do you think?

Reply:
Got it, thank you @Tuubz @sleepylee @lydiasama 🙏